components/service-operator/apis/storage/v1beta1/s3_types.go

```diff
@@ -44,20 +44,17 @@ var (
 		"s3:GetObjectAcl",
 		"s3:GetBucketObjectLockConfiguration",
 		"s3:GetObjectVersionAcl",
-		"s3:HeadBucket",
```

Comment:
HeadBucket is very useful, please don't remove it! ... it's used by the aws sdk to implement all the `Wait*` things

Reply:
The API or the permission?

Reply:
it's the API ... but I swear I've had to add that action before, most confusing.